src/integrationTest/java/uk/gov/hmcts/darts/workflow/command/DeployRedisCommand.java

```
@@ -27,16 +29,36 @@ public void cleanupResources() {
     }
 
     @Override
-    public void execute() {
+    public boolean isSuccess() {
+        return false;
+    }
+
+    @Override
+    public void execute(Map<String, String> envVariables) throws IOException {
+    }
```

Should this have throw new UnsupportedOperationException(); like previous?